_posts/2016-08-14-three-pillars-of-continuous-delivery.md

```diff
@@ -16,32 +16,32 @@ tags:
 
-**Use the right tools.** Identify the tools which can help you. In the modern software development we have plenty of options. Tools like Jenkins, Bamboo, Travis, Ansible, Docker can help you in the automation, but without a good justification of usage they can be just fancy tools. Your automation should evolved together with your needs and problems which are introduced by these problems. Proves of concept will allow you to understand if a specific tool is giving any value in the context of automation.
+**Use the right tools.** Identify the tools which can help you. In the modern world of software development we have plenty of options. Tools like Jenkins, Bamboo, Travis, Ansible, Docker can help you in the automation, but without a good justification of usage they can be just fancy tools. Your automation should evolve together with your needs and problems which are introduced by these problems. Proofs of concept will allow you to understand if a specific tool is adding any value in the context of automation.
 
 ### Rapid Response
 
-**Give answers now.** Continuous deployment should give your answer if your software is valid and ready to use. If your deployment process takes days, weeks or even hours in some cases, you can miss opportunity to get these answers. The time between the discovery and the fix of an issue can be minimised by the rapid response from your continuous deployment process.
+**Give answers now.** Continuous deployment should give your answer if your software is valid and ready to use. If your deployment process takes days, weeks or even hours in some cases, you can miss the opportunity to get these answers. The time between the discovery and the fix of an issue can be minimised by the rapid response from your continuous deployment process.
 
-**Make it faster.**  Many different things can make your process slower. One of this problems is the organisation of your process. In many cases in my previous projects the original version of the process required the individual involvement of many people in the organisation. Sometimes you have to struggle with the slow access to the technical infrastructure in your company. Try to improve these element. The automation of your process should not only include the solutions for technical problems, but it should also improve non-technical aspects of the process.
+**Make it faster.**  Many different things can make your process slower. One such problem is how your processes are organised. In many cases in my previous projects the original version of the process required the individual involvement of many people in the organisation. Sometimes you have to struggle with the slow access to the technical infrastructure in your company. Try to improve these elements. The automation of your process should not only include the solutions for technical problems, but it should also improve non-technical aspects of the process.
```

Comment:
...slow process due to the state of the technical infrastructure....

is this what you meant?